Dockerfile

```diff
@@ -23,6 +23,7 @@ ENV LC_ALL=en_GB.UTF-8 \
 # Configure R
 RUN echo '\n.libPaths("~/R/library")' >> /usr/local/lib/R/etc/Rprofile.site \
     && echo "PATH=\"${PATH}\"" >> /usr/local/lib/R/etc/Renviron \
+    && echo "AWS_DEFAULT_REGION=eu-west-1" >> /usr/local/lib/R/etc/Renviron \
```

Comment:
still not working :cry:

Reply:
I lie... it works in the R session but not in the terminal. Ideally it'd be set in both